hvpy/api.py

```diff
@@ -1,13 +1,76 @@
 __all__ = ["fake_api_call"]
 
+from datetime import datetime
+from typing import Optional
+import requests
+from pydantic import BaseModel, Field
 
-def fake_api_call():
+
+class InputParameters(BaseModel):
+    date: datetime
+    sourceId: int
+    jpip: Optional[bool] = False
+    Json: Optional[bool] = Field(False, alias="json")
+
+
+def parse_response(response: requests.Response, output_parameters: str):
```

Comment:
Let's create io.py and add an enum class OutputType to handle the different output_parameters